File: codeforces/29.calculating_function_[486A]/calculating_function.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  signed long long int n;
  cin >> n;
  if (n != 0) {
    if ((n % 2) != 0) {
      n += 1;
      n = -1 * n;
    }
    n /= 2;
  }
  else {
    n++;
  }
  cout << n << endl;
  return 0;
}

File: codeforces/low_difficulty/difficulty_800/29.calculating_function_[486A]/calculating_function.py
# Author: José Rodolfo (jric2002)
n = int(input())
if (n != 0):
  if ((n % 2) != 0):
    n += 1
    n = -1 * n
  n = int(n / 2)
else:
  n += 1
print(n)